Adding a note for 1.6.0 requirement (#91)
timholy/SnoopCompile.jl: docs/src/snoopr.md

# Snooping on invalidations: `@snoopr`

!!! note

>     Requires `Julia 1.6.0-DEV.154` or above for running, but the result is useful for all versions.

Reviewer: ```suggestion
    `@snoopr` is available on `Julia 1.6.0-DEV.154` or above, but the results can be relevant for all Julia versions.
```
